src/Relations/MorphToMany.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Relations;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+
+class MorphToMany extends BelongsToMany
```

Comment:
It must extend `Illuminate\Database\Eloquent\Relations\MorphToMany` to satisfy `morphToMany` return type.